CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+# Release 0.19.1
+
+# What's New
+
+* Ziti CLI now has 'Let's Encrypt' PKI support to facilitate TLS connections to Controller from
+  BrowZer-based apps that use the `ziti-sdk-js`.
+
+    * New command to Register a Let's Encrypt account, then create and install a certificate
+
+        Usage:
+
+        `ziti pki le create -d domain -p path-to-where-data-is-saved [flags]`
+
+        Flags:
+
+          -a, --acmeserver string                             ACME CA hostname (default "https://acme-v02.api.letsencrypt.org/directory")
```

Comment:
What is the point of having this flag?

Reply:
It's an additional way to specify the ACME server, and partially redundant with the --staging flag.  Upshot is that these are to facilitate using either the LE/ACME staging server, or production server, or some tbd server.  During testing/debugging, it is not uncommon to generate certs many times in a short period of time.  If you do that against the prod ACME server, you will get throttled/blocked/denied.  Odds of that happening when using the staging server are minimal.